apps/api/v2/src/modules/slots/slots-2024-09-04/services/slots.service.ts

```diff
@@ -41,13 +44,14 @@ export class SlotsService_2024_09_04 {
     private readonly teamsRepository: TeamsRepository
   ) {}
 
-  async getAvailableSlots(query: GetSlotsInput_2024_09_04) {
+  async getAvailableSlots<T extends GetSlotsInput_2024_09_04 | GetSlotsInputWithRouting_2024_09_04>(
+    query: T
+  ) {
     try {
       const queryTransformed = await this.slotsInputService.transformGetSlotsQuery(query);
       const availableSlots: TimeSlots = await getAvailableSlots({
         input: {
           ...queryTransformed,
-          routingFormResponseId: queryTransformed.routingFormResponseId ?? undefined,
```

Comment:
We don't need to handle it here now as queryTransformed will have the correct value